### ISSUE TITLE
No autoplay for already watched series on Android TV 

### ISSUE THREAD

**Issue by Cmc303** [Stremio/stremio-web#327]
The ability to unmark episodes as watched on Android TV should be added, it will not autoplay next episode on a series unless the episodes are unwatched.

**Comment by TRtomasz** [comment]
This is wrong repository for this issue please use stremio-bugs for bugs and stremio-features for features
